### PR TITLE
WIP - Cilium CLI tests

### PR DESCRIPTION
**Pull request by giorio94**

### CODE REVIEW

File: .github/workflows/conformance-e2e.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ concurrency:
 env:
   # renovate: datasource=github-releases depName=cilium/cilium-cli
   cilium_cli_version: v0.15.10
```
Comment (brb) [inline .github/workflows/conformance-e2e.yaml]: Unfortunately, you need to `s/v0.15.10//`, as it takes precedence over the `cli_ci_version` - https://github.com/cilium/cilium-cli/blob/main/action.yaml#L64.

Comment (giorio94) [reply .github/workflows/conformance-e2e.yaml]: Ah, I thought the precedence was the other way round. Fixed now

Comment (giorio94) [reply .github/workflows/conformance-e2e.yaml]: Thanks!

**Pull request closed without merging**